spec/mailers/user_mailer_spec.rb

```diff
@@ -28,7 +28,11 @@
 describe UserMailer, type: :mailer do
   before(:each) do
     @ac = mock_app_config(admin_email: 'admin@email.com',
-                          disable_user_emails: false)
+                          disable_user_emails: false,
```

Comment:
Oh wait, is it from switching from `spy` to `instance_spy`? Do you remember why we did this?